### DISCUSSION TITLE
Drop down budget select on "no-budget" transaction list

### DISCUSSION THREAD

**Discussion by creeg2** [firefly-iii/firefly-iii#6380] [General]
In order to make assigning transactions to a budget a one-click action, it would be nice if on the "no-budget" list, the available budgets could be selected in a drop down for each transaction in the table.  The table would then either auto refresh, or have a "update transactions" button on the bottom.

Right now, to assign a transaction to a budget takes:
0. From the "no-budget" list
1. Click on transaction description
2. Click "actions"
3. Click "edit"
4. Select budget (2 clicks)
5. Click "update transaction"
At this point you're dropped back into the transaction summary page, so now you have to go back to the no-budget list
6. Click 'budgets"
7. Click "expenses without budget"
8. Go to 1

As it is now, unless I'm missing something, it takes 8 clicks to repeatedly assign transactions to budgets.  Ideally it would just be one click.  Obviously rules auto-assigning budgets are a better option than manual entry, but this can not be done for every transaction.

**Comment by JC5** [comment]
Good idea, I'll join it with https://github.com/firefly-iii/firefly-iii/issues/6179

**Reply by creeg2** [reply]
Thanks!
